### PR TITLE
Remove PUT from FW

### PR DESCRIPTION
**Pull request by Hagridpon3**
Remove PUT + uartslave

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -495,7 +495,7 @@ target_sources(
           $<$<STREQUAL:${FILAMENT_SENSOR},YES>:src/common/filament_sensor_photoeletric.cpp>
           $<$<BOOL:${FILAMENT_SENSOR}>:src/common/thread_measurement.cpp>
           src/common/hardware_serial.cpp
-          src/common/diag.cpp
+          # src/common/diag.cpp
```
Comment (dragomirecky) [inline CMakeLists.txt:498]: No commented lines. Please remove it instead ☺️

File: src/common/appmain.cpp [hunk 2]
```diff
@@ -20,11 +20,10 @@
     #include "sim_heater.h"
 #endif //SIM_HEATER
 
-#include "uartslave.h"
+//#include "uartslave.h"
```
Comment (dragomirecky) [inline src/common/appmain.cpp:23]: please remove instead

File: src/common/appmain.cpp [hunk 3]
```diff
@@ -63,7 +62,7 @@ extern void reset_trinamic_drivers();
 extern "C" {
 
 extern uartrxbuff_t uart6rxbuff; // PUT rx buffer
-extern uartslave_t uart6slave;   // PUT slave
+//extern uartslave_t uart6slave;   // PUT slave
```
Comment (dragomirecky) [inline src/common/appmain.cpp:65]: please remove instead

File: src/common/appmain.cpp [hunk 4]
```diff
@@ -112,21 +106,10 @@ void app_run(void) {
...
-                hwio_fan_set_pwm(i, 0); // disable fans
-        }
-        if (INIT_TRINAMIC_FROM_MARLIN_ONLY == 0) {
-            init_tmc();
-        }
-        reset_trinamic_drivers();
-    } else {
-        app_setup();
-        marlin_server_start_processing();
-    }
```
Comment (dragomirecky) [inline src/common/appmain.cpp:129]: 😍 happy to see this go away

File: src/gui/guimain.cpp [hunk 5]
```diff
@@ -118,8 +117,8 @@ void client_gui_refresh() {
 }
 
 void gui_run(void) {
-    if (diag_fastboot)
-        return;
+    //if (diag_fastboot)
+    //    return;
```
Comment (dragomirecky) [inline src/gui/guimain.cpp:122]: please remove instead

**Review by RadekVana — APPROVED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (Hagridpon3) [reply CMakeLists.txt]: Fixed

File: src/common/appmain.cpp (hunk 2 above)
Comment (Hagridpon3) [reply src/common/appmain.cpp]: Fixed

File: src/common/appmain.cpp (hunk 3 above)
Comment (Hagridpon3) [reply src/common/appmain.cpp]: Fixed

File: src/gui/guimain.cpp (hunk 5 above)
Comment (Hagridpon3) [reply src/gui/guimain.cpp]: Fixed

**Review by dragomirecky — DISMISSED** [review]

**Review by dragomirecky — APPROVED** [review]

**Pull request merged**